cas-server-support-jpa-ticket-registry/src/test/resources/jpaTicketRegistryContext.xml

```diff
@@ -108,6 +108,12 @@
         </tx:attributes>
     </tx:advice>
 
+    <tx:advice id="txRegistryProxyGrantingTicketDelegatorAdvice" transaction-manager="ticketTransactionManager">
```

Comment:
You also want to update the main context file that is pulled in. Same sort of changes should be applied. It's under META-INF/Spring.